Dockerfile

```diff
@@ -1,26 +1,24 @@
 # Copyright (c) 2021 LG Electronics Inc.
 # SPDX-License-Identifier: Apache-2.0
-FROM	ubuntu:20.04
+FROM --platform=linux/amd64 python:3.8-slim-buster
 
-RUN     apt-get clean
-RUN 	apt-get update && apt-get install sudo -y
-RUN	ln -sf /bin/bash /bin/sh
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
`noninteractive` mode is useful when we need zero interaction while installing or upgrading the system via apt. However, setting this environment variable to noninteractive has an adverse effect. Because it is inherited by all images and containers built from the image, effectively changing their behavior.

```suggestion
```

Let's look for an alternative.